pages/_admin/index.tsx

```diff
@@ -0,0 +1,106 @@
+import Head from 'next/head'
+import {Button} from '@chakra-ui/react'
+import { providers, signIn, signOut, useSession, getSession } from 'next-auth/client'
+
+import styles from '../styles/Home.module.css'
+import StoryForm from '../../components/form'
+
+import prisma from '../../lib/prisma'
+
+export default function _Admin({ stories, providerList }) {
+  const [ session ] = useSession()
+  return <>
+    {!session && <>
+      Not signed in <br/>
+    
+      {Object.values(providerList).map(provider => (
+        <div key={provider.name}>
+          <button onClick={() => signIn(provider.id)}>Sign in with {provider.name}</button>
+        </div>
```

Comment:
`provider.name`, `provider.id` are throwing TS errors `Property 'name' does not exist on type 'unknown'`

Reply:
Maybe we need types? `npm install --save-dev @types/next-auth`